addon-test-support/wait-until.js

```diff
@@ -3,13 +3,8 @@ import RSVP from 'rsvp';
 
 export function waitUntil(callback, { timeout = 1000 } = {}) {
   return new RSVP.Promise(function(resolve, reject) {
-    let value = run(callback);
-    if (value) {
-      resolve(value);
-      return;
-    }
     let time = 0;
-    let tick = function() {
+    function tick() {
```

Comment:
If I'm right, you should schedule the first tick with a 0 ms delay to exit as soon as possible but still respond asynchronously, correct?

Reply:
Yes, confirm. We need to change the initial time value to -10 and change the first tick to use 0. That’s what I did in my branch in the test helpers.

Reply:
Can you port that?

Reply:
Yep, absolutely. Was just trying to land it in test helpers then bring back any changes/tweaks from the implementations here once the PR in ember-test-helpers has landed.